scripts/removeUnusedResourcesFromAssets.swift

```diff
@@ -1,114 +0,0 @@
-import Foundation
```

Comment:
why?

Reply:
This was accidentally removed, just added it back in the latest commit.